crates/noirc_frontend/src/hir/mod.rs

```diff
@@ -3,6 +3,7 @@ pub mod def_map;
 pub mod resolution;
 pub mod scope;
 pub mod type_check;
+pub mod visilibity;
```

Comment:
```suggestion
pub mod visibility;
```

This module name has a typo

Reply:
Done